.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: rust
-rust: nightly
-script: cd test-project && cargo test
+rust:
+ - stable
+ - nightly
+matrix:
+ include:
+   - rust: stable
+     env: FEATURES="--features stable"
+   - rust: nightly
```

Comment:
Please also add a beta build. This is hugely helpful to Rust in catching beta regressions (https://www.reddit.com/r/rust/comments/600mwc/psa_hey_rust_users_especially_library_authors/).

Reply:
Added.